DataDefinitions/Crime.cs

```diff
@@ -37,6 +37,12 @@ static Crime()
             var PassengerWanted = new Crime("passengerWanted");
         }
 
+        // Faction report definition
```

Comment:
There's a subtle gotcha here (I mention it in docs\Localization_for_devs.md. Any instances that you want to expose publically should be declared here but *must* be inititalized in the class initialiser above, so that `AllOfThem` gets correctly populated from the get-go. Otherwise all kinds of wierdness occurs. I'll amend.